api/sheet.go

```diff
@@ -46,6 +47,12 @@ type SheetVCSPayload struct {
 	LastSyncTs   int64  `json:"lastSyncTs"`
 }
 
+// SheetTabPayload is the API message payloads for UI tab context in SQL editor.
+// The sheet source should be SheetFromBytebase.
+type SheetTabPayload struct {
```

Comment:
@LiuJi-Jim @d-bytebase I am not sure we discussed this before. I can't remember why we need to persist in this state in the database. What's the issue with storing it in local storage?

Reply:
+1 for pinia store.